test/common/common.go

```diff
@@ -8,6 +8,8 @@ import (
 	"reflect"
 	"time"
 
+	"github.com/antihax/optional"
```

Comment:
Is this a library they already use?

Reply:
ohh wait good question... I will double check this.

Reply:
Yes have a look at https://github.com/mrgunior/osb-checker/blob/dev/autogenerated/go-client/api_service_instances.go